### DISCUSSION TITLE
Notion DB Loader

### DISCUSSION THREAD

**Discussion by Darksheer2017** [langflow-ai/langflow#250] [Ideas]
It would be great, if the Notion DB Loader could be implemented as an onjectz into LangFlow: https://python.langchain.com/en/latest/modules/indexes/document_loaders/examples/notiondb.html

Is there any guidance on how to provide support regarding writing such integrations/implementations?

I'd even spnsor someone doing this work (and a couple more integrations as well), if pointed into the right direction. Thanks!

**Comment by ogabrielluiz** [comment]
Hey, @Darksheer2017. 

The process involves editing the [config.yml](https://github.com/logspace-ai/langflow/blob/dev/src/backend/langflow/config.yaml) to include the loaders you require and testing them inside LangFlow.

**Reply by lawyerupau** [reply]
I tried to do this, by adding in the following to my fork but it doesnt seem to do anything?

```
  NotionDB:
    documentation: "https://python.langchain.com/docs/modules/data_connection/document_loaders/integrations/notiondb" 
```

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
